<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NordVPN Front-End</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>231</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectBut">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconect</string>
    </property>
   </widget>
   <widget class="QLabel" name="setting1_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Technology</string>
    </property>
   </widget>
   <widget class="QLabel" name="setting2_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Protocol</string>
    </property>
   </widget>
   <widget class="QComboBox" name="techCombo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="protoCombo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="KillSwitch">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Kill Switch</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CyberSec">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>CyberSec</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ofs">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ofuscate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Notify">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Notify</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutoC">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Connect</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dns">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>DNS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Carian</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Server Location</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QTreeView" name="groups">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titleimage">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="nordvpnfront.qrc">:/nordvpn.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="nordvpnfront.qrc"/>
 </resources>
 <connections/>
</ui>
